components/formats-gpl/src/loci/formats/in/ZeissCZIReader.java

```diff
@@ -88,6 +89,11 @@
 
   // -- Constants --
 
+  private static final String ALLOW_AUTOSTITCHING_KEY = "zeissczi.allow.autostitch";
```

Comment:
Semantics question: should we agree on some minimal naming schema for defined options? @joshmoore used `classname.key` in https://github.com/openmicroscopy/bioformats/pull/2013 although the latter referred directly to a field of the class.